src/StateService/StateService/config.json

```diff
@@ -3,7 +3,9 @@
     "Path": "Data_MPT_{0}",
     "FullState": false,
     "Network": 860833102,
-    "AutoVerify": false
+    "AutoVerify": false,
+    "MaxFindResultItems": 100,
+    "MaxPageNumber": 100
```

Comment:
What happens if there is a find query with over 100x100 results?

Reply:
This is the default settings, if you want more, you can change it.